datafusion/substrait/src/logical_plan/producer.rs

```diff
@@ -2310,39 +2306,6 @@ mod test {
         Ok(())
     }
 
-    #[test]
-    fn custom_type_literal_extensions() -> Result<()> {
```

Comment:
sadly as we no longer produce custom types, there's no easy way to test for them either :/ 
maybe if we one day implement some UDT support then that can be used for testing here instead. But for now there's no good replacement.